packages/angular/src/search-token-server/files/scripts/port-allocator.mjs

```diff
@@ -81,9 +81,11 @@ const isPortAvailable = async (port, options) => {
 
 function updateAppPort(port) {
   const angularJsonPath = resolve('angular.json');
+  const packageJsonPath = resolve('package.json');
```

Comment:
No more projectDefault. Sadge. Using package name instead, same story in the end